redirects.yaml

```diff
@@ -149,6 +149,10 @@ docs/users-sample-commands: /docs/sample-command-usage-and-output-interpretation
 docs/users: /docs/user-types-and-abilities
 docs/whats-new: /docs/whats-new-2-6
 
+# Removal of all docs
```

Comment:
I'm not sure how these are applied, but if it's in order this should probably go under the last `docs` redirect on ln 160